tensorboard/tools/inline_file_content.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Inline file contents in destination file.
+
+Replaces special template syntax (e.g., `%filename.ext%`) in the destination file.
+"""
+
+import os
+import sys
+
+
+def inline_content():
```

Comment:
Slightly more conventional to call this `main()`.  Also can you provide (here or in module doc) a brief example of the CLI arguments?  E.g.  `usage: inline_file_content.py <dest-file> <src-file> [<src-file>...]`